src/client.cpp

```diff
@@ -789,8 +790,44 @@ void CClient::OnHandledSignal ( int sigNum )
 #endif
 }
 
-void CClient::OnControllerInFaderLevel ( int iChannelIdx, int iValue )
+// Ensure user's own channel is always given MIDI offset 0, so it is the first physical fader
+// All channels with a client ID less than user's own will use ID+1 as their MIDI offset
+
+int CClient::ChanToMIDI ( const int iChannelIdx )
+{
+    if ( iMyChannelID == INVALID_INDEX )
```

Comment:
As above, it was suggested using `INVALID_INDEX` for something that isn't an index into a vector wasn't a good idea.

Reply:
See my PR.